code/modules/mob/living/basic/stoats/stoat.dm

```diff
@@ -0,0 +1,47 @@
+/mob/living/basic/stoat
+	name = "stoat"
+	desc = "A natural born vermin exterminator... a janitor's best friend."
+	icon_state = "stoat"
+	icon_living = "stoat"
+	icon_dead = "stoat_dead"
+	base_icon_state = "stoat"
+	icon = 'icons/mob/simple/pets.dmi'
+	butcher_results = list(/obj/item/food/meat/slab = 1)
+	mob_biotypes = MOB_ORGANIC
+	mob_size = MOB_SIZE_SMALL
+	health = 50
+	maxHealth = 50
+	melee_damage_lower = 6
+	melee_damage_upper = 9
+	verb_say = "whiskers"
```

Comment:
this is a little pedantic but I don't think `whiskers` works as a replacement for `says`?
`whickers` is a word but it's more sort of a thing a horse would do